src/tdd/Constants.java

```diff
@@ -0,0 +1,8 @@
+package tdd;
```

Comment:
Look at proposed changes in attachment, tests could have to be adjusted to account for corrected totals.
[Constants.txt](https://github.com/zachj1582/tdd/files/4963235/Constants.txt)